playground/mocks/spec-okta-api/idp/idx/index.js

```diff
@@ -1,5 +1,9 @@
 const templateHelper = require('../../../config/templateHelper');
 const cancelTransaction = require('../../../data/idp/idx/identify-with-no-sso-extension');
+// const biometricsErrorMobile = 
```

Comment:
remove

Reply:
@pradeepdewda-okta this change is related to below template change. I would like to inform later developers how to test the flow in playground. Is there any existing way that i can make it work without adding a comment?

Reply:
you can put right above the `template` variable and say use this file for other two errors..
```
error-okta-verify-uv-fastpass-verify-enable-biometrics-mobile
error-okta-verify-uv-fastpass-verify-enable-biometrics-desktop
```